saplings/contracts/public/manifest.json

```diff
@@ -6,4 +6,3 @@
   "theme_color": "#000000",
   "background_color": "#ffffff"
 }
-  
```

Comment:
This commit needs a longer message to describe what its doing. However, it looks like it is just removing a trailing newline. Every file committed to Github should have this trailing newline for editor compatibility. Github helps you out by displaying that red circled "-" if you accidentally committed a file without a trailing newline.